packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -357,45 +343,16 @@ abstract class ServerlessClusterBase extends Resource implements IServerlessClus
  * @resource AWS::RDS::DBCluster
  *
  */
-export class ServerlessCluster extends ServerlessClusterBase {
+abstract class ServerlessClusterNew extends ServerlessClusterBase {
 
-  /**
-   * Import an existing DatabaseCluster from properties
-   */
-  public static fromServerlessClusterAttributes(scope: Construct, id: string,
-    attrs: ServerlessClusterAttributes): IServerlessCluster {
+  protected readonly newCfnProps: CfnDBClusterProps;
+  protected readonly subnetGroup: ISubnetGroup;
```

Comment:
Why is this a field, and protected? I don't see it used anywhere outside of this class' constructor?

Reply:
I copied this line from `cluster.ts`. You're right. I was able to remove field `subnetGroup`.